keps/sig-node/1769-memory-manager/README.md

```diff
@@ -643,24 +745,28 @@ details). For now, we leave it here.
 
 _This section must be completed when targeting beta graduation to a release._
 
-* **How does this feature react if the API server and/or etcd is unavailable?**
+###### How does this feature react if the API server and/or etcd is unavailable?
   No impact.
 
-* **What are other known failure modes?**
+###### What are other known failure modes?
+
```

Comment:
There is also the failure to restart the kubelet on disabling of the feature.

Reply:
Added (copied), thanks.